src/utils.js

```diff
@@ -56,7 +56,32 @@ function logError(msg) {
   log(msg, errorNamespace)
 }
 
+function decrementHugeNumberBy1(n) {
```

Comment:
Please add a test for this function.
Sorry by I don't understand what it does :D

Reply:
Ok, this function is for fetching next page's tweets param `max_id`, we have to decrease 1 for fetching data that not repeated with previous data.